website/docs/layouts/grid/partials/code/how-to-use.md

```diff
@@ -140,11 +140,19 @@ If the grid items are wrapping on multiple lines, you have to overwrite both the
 </Hds::Layout::Grid>
 ```
 
+## Column width management
+
 ### Column min width
 
-Specify a `columnMinWidth` size to exercise control over how many columns occupy a row. If the total widths of the columns add up to more than 100% of the parent they will automatically wrap to the next row as necessary to fit.
+!!! Info
```

Comment:
[suggestion] Not sure we need a banner for this, probably the two paragraphs could be combined/adapted so they're part of the normal text. What do you think?

Reply:
mmm, personally it didn't come through, when reading it. @andrew-hashicorp since you've recently worked on standardization of banners, what do you think?

Reply:
I can rewrite it

Reply:
Ok, I just though highlighting it would help consumers better see it and be easier able to read at a glance so they don't gloss over it or have to read over everything else to figure out what they need.

I can update it though.

Reply:
Ok I updated it and reformatted into an intro section which I think works ok. Please see what you think though. @andgen404 @didoo

Reply:
works for me

Reply:
Thanks!